### PR TITLE
Kill old nodes in test-runner (optional)

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR addresses me.

Proposed changes include:
* Tired of running `pkill beam`?
* Not sure if it's up or down?
* Irritated that nodes are kept running, if big-tests fails?
* This PR adds `--kill-old-nodes` argument, that stops already running nodes before compiling and starting any other nodes.
* No other erlang node gets hurt (only MongooseIM ones).
* Some documentation included.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[6503.1](https://travis-ci.org/esl/MongooseIM/jobs/533278881) / Erlang 19.3 / small_tests / 048ff9192b59da0640197e5c62655f96cb15661d
Reports [root](http://esl.github.io/mongooseim-ct-reports/s3_reports.html?prefix=PR/2306/6503/small_tests.19.3) / [small](https://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/PR/2306/6503/small_tests.19.3/small/ct_run.mongooseim@localhost.2019-05-16_12.21.45/lib.mongooseim.logs/run.2019-05-16_12.21.46/suite.log.html)

**Comment by arcusfelis** [comment]
I've skipped CI for the last commit, because it changes file, not executed by Travis in any way.

**Comment by DenysGonchar** [comment]
does it work with `--tls-dist` option? i guess no.

File: tools/test-runner.sh [hunk 1]
```diff
@@ -104,6 +104,10 @@ Script examples:
     Sets dev-nodes and test-hosts to empty lists
     Reruns mam_SUITE
 
+./tools/test-runner.sh --preset mysql_redis --skip-small-tests --skip-cover --one-node --kill-old-nodes -- rdbms
```
Comment (fenek) [inline tools/test-runner.sh]: The new switch is not described in usage instructions earlier in this file.

File: tools/test_runner/kill_nodes.erl [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    main([]).
+
+main(Opts) ->
+    Nodes = opts_to_nodes(Opts),
+    Result = kill_nodes(Nodes),
+    io:format("kill_nodes: done ~p~n", [Result]),
+    erlang:halt().
+
+kill_nodes(Nodes) ->
+    [{Node, rpc:call(Node, init, stop, [])} || Node <- Nodes].
```
Comment (fenek) [inline tools/test_runner/kill_nodes.erl]: Won't be really helpful if the nodes are frozen. But should work in most cases. :)

**Comment by arcusfelis** [comment]
We will kill them later.

**Pull request closed without merging**